src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDoclet.java

```diff
@@ -444,7 +421,29 @@ public Set<? extends Option> getSupportedOptions() {
         return configuration.getOptions().getSupportedOptions();
     }
 
-    private void performCopy(String filename, DocPath targetPath) throws DocFileIOException {
+    private void copyResource(DocPath sourcePath, DocPath targetPath, boolean replaceNewLine)
+            throws DocletException {
+        DocPath resourcePath = DocPaths.RESOURCES.resolve(sourcePath);
+        // Look for resource in formats/html resources first
+        URL resourceURL = HtmlConfiguration.class.getResource(resourcePath.getPath());
+        // If not found try toolkit resources
+        if (resourceURL == null) {
+            resourceURL = BaseConfiguration.class.getResource(resourcePath.getPath());
```

Comment:
It always seemed wrong (a bug) that `stylesheet.css` was down in the `toolkit` world.
It is also possible to reason that `script.js` should not be in the `toolkit` world either.
Maybe this would be a good time to move those files into the `formats/html` world, as
well as the two `.svg` files, so that all resource files that need to be copied into the 
generated API directories are in the `formats/html` world.

In other words, we might be able to avoid lines 430-432

Reply:
Yes, that occurred to me as well. I'll move the files and update the code.